jsonnet/components/telemeter-client.libsonnet

```diff
@@ -28,6 +28,12 @@ function(params) {
   secret: tc.telemeterClient.secret,
   servingCertsCABundle: tc.telemeterClient.servingCertsCABundle,
   deployment: tc.telemeterClient.deployment {
+    metadata+: {
+      labels+: {
+        'app.kubernetes.io/managed-by': 'cluster-monitoring-operator',
+        'app.kubernetes.io/part-of': 'openshift-monitoring',
```

Comment:
Same comment as the one for openshift-state-metrics.

Reply:
👍